### PR TITLE
Add explanation for empty email and announcement.

### PR DESCRIPTION
**Pull request by justin-time**
This little PR adds a note to the email/announcement window that the user has
to connect the server to view the messages.

### CODE REVIEW

File: message.cpp [hunk 1]
```diff
@@ -27,7 +33,10 @@ void message::updateDate(QString date)
 
 void message::updateMessage(QString body)
 {
-    ui->message_body->setText(body);
+    if (body.isEmpty())
+        ui->message_body->setText(tr("Bitte synchronisiere dich mit dem L²P, um die Nachricht zu lesen!"));
```
Comment (Sync-my-L2P) [inline message.cpp]: Suggestion: Tell the user that he has to log in instead of to synchronise.

**Comment by justin-time** [comment]
I'm not sure, because you can log in without synchronising and this would not work, too…

**Comment by Sync-my-L2P** [comment]
That's true. I totally forgot about this possibility.
Nevertheless the typical user could be confused by the word "synchronise". He could think he has to download the announcement by pressing the download button.
Another suggestion: The user has to "connect to l2p".

**Comment by justin-time** [comment]
Ok, that's really better! 

Thanks for merging!

**Pull request merged**